### PR TITLE
Adding a helper to set the name of the MCParticle collection

### PR DESCRIPTION
**Pull request by jstrube**
Following guidelines laid out by @andresailer, I'm attempting to add a helper to allow setting the name of the MCParticle collection to something other than "MCParticle".

BEGINRELEASENOTES
- allow to change the name of the MCParticle input collection in ddsim
        - using parameter 'mcParticleCollectionName'

ENDRELEASENOTES

### CODE REVIEW

File: DDSim/DDSim/Helper/MCParticles.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    super(MCParticles, self).__init__()
+    self._parameters["MCParticleCollectionName"] = "MCParticle"
+
+  @property
+  def mcParticleCollectionName( self ):
+    """Set the name of the collection containing the MCParticle input.
+    Default is "MCParticle".
+    """
+    return self._parameters["MCParticleCollectionName"]
+  @particlesPerEvent.setter
```
Comment (andresailer) [inline DDSim/DDSim/Helper/MCParticles.py]: This needs to be renamed as well.

**Comment by andresailer** [comment]
Related to aidasoft/DD4hep#147

You need to add the corresponding calls for your addition:

Like what is done for GuineaPig in DD4hepSimulation

https://github.com/jstrube/lcgeo/blob/ea188ef7f904ad5edab709ee295ddadee372df6a/DDSim/DDSim/DD4hepSimulation.py#L45 

https://github.com/jstrube/lcgeo/blob/ea188ef7f904ad5edab709ee295ddadee372df6a/DDSim/DDSim/DD4hepSimulation.py#L87

https://github.com/jstrube/lcgeo/blob/ea188ef7f904ad5edab709ee295ddadee372df6a/DDSim/DDSim/DD4hepSimulation.py#L376

Also please rename `MCParticle` to `LCIO`

File: DDSim/DDSim/Helper/LCIO.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+"""Helper object for files containing one or more MCParticle collections"""
+
+from DDSim.Helper.Input import Input
+
+class MCParticles( Input ):
```
Comment (andresailer) [inline DDSim/DDSim/Helper/LCIO.py]: Sorry I wasn't clear, the class should also be called LCIO.
Then you get the "--LCIO.mcParticleCollectionName" command line flag and option named "LCIO...." in the steering file.

Conceivably we get more options for lcio files at some point.

**Comment by gaede** [comment]
@jstrube are still working on addressing the issues ? Would be very useful to have in ...

**Comment by jstrube** [comment]
I thought this is done now. What's still missing? Did I not address Andre's suggestions?

File: DDSim/DDSim/Helper/LCIO.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    super(MCParticles, self).__init__()
+    self._parameters["MCParticleCollectionName"] = "MCParticle"
+
+  @property
+  def mcParticleCollectionName( self ):
+    """Set the name of the collection containing the MCParticle input.
+    Default is "MCParticle".
+    """
+    return self._parameters["MCParticleCollectionName"]
+  @particlesPerEvent.setter
```
[the remark this answers is not in the document]
Comment (jstrube) [reply DDSim/DDSim/Helper/LCIO.py]: Not sure I understand what's still missing. Does this diff address your comment?

**Comment by jstrube** [comment]
Sorry, I didn't get what the problem was. I think this is fixed now.

**Comment by gaede** [comment]
@jstrube I have added release notes for you. Please do not remove the BEGINRELEASENOTES/ENDRELEASENOTES tags but rather fill in what you changed in future PRs

**Comment by jstrube** [comment]
I thought that's what I did in the first comment in this thread?
Did the additional commits screw this up?

**Pull request merged**